### PR TITLE
Log insights.chrome.navigation usage

### PR DESCRIPTION
**Pull request by Hyperkid123**
jira: https://issues.redhat.com/browse/RHCLOUD-9190

I have found only one deprecated function.

@karelhala @ryelo are you aware of some other methods that should be deprecated?

### CODE REVIEW

**Review by karelhala — APPROVED** [review]
I don't think there are other functions.

**Comment by Hyperkid123** [comment]
@karelhala is the sentry usage ok? I have never used it before. I assume account info is passed to it by default right?

File: src/js/chrome/entry.js [hunk 1]
```diff
@@ -57,7 +58,11 @@ export function chromeInit(navResolver) {
 
   return {
     identifyApp: (data) => navResolver.then(() => identifyApp(data, store.getState().chrome.globalNav)),
-    navigation: () => console.error("Don't use insights.chrome.navigation, it has been deprecated!"),
+    navigation: () => {
+      log('Deprecated function call: insights.chrome.navigation');
+      Sentry.captureMessage('Deprecated function call: insights.chrome.navigation');
+      console.error("Don't use insights.chrome.navigation, it has been deprecated!");
```
Comment (ryelo) [inline src/js/chrome/entry.js:64]: You don't have to do two of these. If you just throw a `console.error` it'll send to sentry automatically

**Review by ryelo — CHANGES_REQUESTED** [review]

File: src/js/chrome/entry.js (hunk 1 above)
Comment (Hyperkid123) [reply src/js/chrome/entry.js:64]: Ah so just `console.error("Don't use insights.chrome.navigation, it has been deprecated!");` is enough? Ok, so I guess no changes are required for this part. It was like that before.

File: src/js/chrome/entry.js [hunk 2]
```diff
@@ -57,7 +57,9 @@ export function chromeInit(navResolver) {
 
   return {
     identifyApp: (data) => navResolver.then(() => identifyApp(data, store.getState().chrome.globalNav)),
-    navigation: () => console.error("Don't use insights.chrome.navigation, it has been deprecated!"),
+    navigation: () => {
```
Comment (karelhala) [inline src/js/chrome/entry.js:60]: No need to wrap in function body.

**Review by karelhala — COMMENTED** [review]
I think we can just leave the documentation and sentry will handle the rest.

**Review by karelhala — APPROVED** [review]

**Pull request merged**